docs/README.md

```diff
@@ -279,14 +279,16 @@ Check out the [**Product Summary documentation**](https://vtex.io/docs/component
+| ------------------------ | --------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
+| `layout`                 | `Enum`    | Whether the Filter Navigator layout should be responsive (`responsive`) or not (`desktop`). You may use `desktop` when the Filter Navigator was configured to be displayed in a [drawer](https://vtex.io/docs/components/content-blocks/vtex.store-drawer@0.9.0). | `responsive`  |
+| `maxItemsDepartment`     | `number`  | Maximum number of departments to be displayed before the See More button is triggered.                                                                                                                                                                            | `8`           |
+| `maxItemsCategory`       | `number`  | Maximum number of category items to be displayed before the See More button is triggered.                                                                                                                                                                         | `8`           |
+| `initiallyCollapsed`     | `Boolean` | Makes the search filters start out collapsed (`true`) or open (`false`).                                                                                                                                                                                          | `false`       |
+| `filtersTitleHtmlTag`    | `string`  | HTML tag for the filter's title.                                                                                                                                                                                                                                  | `h5`          |
+| `scrollToTop`            | `enum`    | Scrolls the page to the top (`auto` or `smooth`) or not (`none`) when selecting a facet.                                                                                                                                                                          | `none`        |
+| `fullWidthOnMobile`      | `Boolean` | Defines if the `filter-navigator` will occupy the entire length of the window on mobile (`true`) or not (`false`)                                                                                                                                                 | `false`       |
+| `navigationTypeOnMobile` | `Enum`    | Defines the navigation method on Mobile. `page` or `collapsible`                                                                                                                                                                                                  | `page`        |
```

Comment:
```suggestion
| `fullWidthOnMobile`      | `boolean` | Whether the `filter-navigator.v3` will be rendered on mobile using the screen full width (`true`) or not (`false`).  | `false`       |
| `navigationTypeOnMobile` | `enum` | Defines how mobile users should navigate on the filter selector component. Possible values are: `page` or `collapsible`. | `page`        |
```

Reply:
It would be nice to add what `page` and `collapsible` mean in practice.

Reply:
Done! What do you think?